modules/rhcos-enable-node-scan-scannerv4.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities/scan-rhcos-node-host.adoc
+// * cloud_service/upgrading-cloud/upgrade-cloudsvc-roxctl.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rhcos-enable-node-scan-scannerv4_{context}"]
+= Enabling {op-system} node scanning with Scanner V4
+
+[role="_abstract"]
+If you use {ocp}, you can enable scanning of {op-system-first} nodes for vulnerabilities by using {rh-rhacs-first}.
+
+:FeatureName: RHCOS node scanning with Scanner V4
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+* For scanning {op-system} node hosts of the secured cluster, you must have installed the following:
+** Secured Cluster services on {ocp} {ocp-supported-version} or later
+** {product-title-short} version 4.6 or later
++
+For information about supported platforms and architecture, see the link:https://access.redhat.com/articles/7045053[{product-title} Support Matrix]. For life cycle support information for {product-title-short}, see the link:https://access.redhat.com/support/policy/updates/rhacs[{product-title} Support Policy].
+
+.Procedure
+
+To enable node indexing, also known as node scanning, by using Scanner V4:
+
+. In the Central pod, on the `central` container, set the `ROX_NODE_INDEX_ENABLED` variable to `true` by running the following command on the Central cluster:
++
+[source,terminal]
+----
+$ kubectl -n stackrox set env deployment/central ROX_NODE_INDEX_ENABLED=true
```

Comment:
we usually show both `oc` and `kubectl`, right? Should we do that here, too? if so, maybe just show the `oc` command right above this (same exact thing except it's `oc` instead of `kubectl`